### PR TITLE
Fix more issues if re-exporting DI schema

### PR DESCRIPTION
**Pull request by Pro**
Related to: https://github.com/FreeOpcUa/opcua-modeler/issues/9

Work in Progress, please don't merge yet.

### CODE REVIEW

File: opcua/common/xmlexporter.py [hunk 1]
```diff
@@ -125,10 +125,12 @@ def _add_node_common(self, nodetype, node):
                                 BrowseName=browsename,
                                 NodeId=nodeid)
         if parent is not None:
-            node_el.attrib["ParentNodeId"] = parent.nodeid.to_string()
+            node_class = parent.get_node_class()
+            if node_class is ua.NodeClass.Object or node_class is ua.NodeClass.Variable or node_class is ua.NodeClass.Method:
```
Comment (oroulet) [inline opcua/common/xmlexporter.py]: If node_class in (xxxx, xxxx,xxxx):

Is easier to read

Comment (oroulet) [inline opcua/common/xmlexporter.py]: Otherwise all good

**Comment by oroulet** [comment]
I had some not commited changed in that file that I pushed to master. you should try to rebase asap. Now I will have a look at the UI if I get some time...

**Comment by Pro** [comment]
@oroulet I fixed a few more bugs, so this is now ready to merge.

Apart from the wrong namespace ID there are still some minor issues

**Pull request merged**